test/kokoro/xds.sh

```diff
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+set -exu -o pipefail
+if [[ -f /VERSION ]]; then
+  cat /VERSION
+fi
+
+cd github
+
+export GOPATH=~/gopath
```

Comment:
Does `~` always work?
I vaguely remember in race cases it doesn't. And it's expended to  `${HOME}` anyway, it's preferred to use `${HOME}` directly.

Reply:
Done